lambda_code/tests/test_model_utils.py

```diff
@@ -29,3 +35,20 @@ def test_nhs_number_mod11_check(self):
 
         for invalid_nhs_number in invalid_nhs_numbers:
             self.assertFalse(nhs_number_mod11_check(invalid_nhs_number))
+
+    def test_get_nhs_number_verification_status_code(self):
+        """Test the get_nhs_number_verification_status_code function"""
+        # The NHS number verification status code is 01
+        nhs_number_verification_status_code = get_nhs_number_verification_status_code(self.json_data)
+        self.assertEqual(nhs_number_verification_status_code, "01")
+
+        self.json_data["contained"][1]["identifier"][0]["extension"][0]["valueCodeableConcept"]["coding"][0][
```

Comment:
I think I've seen this piece of code in multiple places. If that `code` and it's value `04` has special meaning then, it makes sense to create a function somewhere.